apps/openmw/mwrender/objects.cpp

```diff
@@ -109,6 +110,9 @@ void Objects::insertGrass(MWWorld::CellStore* cell)
 
         insert->setAttitude(makeObjectOsgQuat(grass.mPos));
 
+        osg::ref_ptr<SceneUtil::LightListCallback> lightListCallback = new SceneUtil::LightListCallback;
+        insert->addCullCallback(lightListCallback);
```

Comment:
Definitely a cull callback, not an update callback?

Reply:
A quote from the lightmanager.hpp:

> To receive lighting, objects must be decorated by a LightListCallback. Light list callbacks must be added via
**node->addCullCallback(new LightListCallback).**
 Once a light list callback is added to a node, that node and all its child nodes can receive lighting.

Reply:
Cool. I'd misremembered it as an update callback for some reason, but that obviously wouldn't work very well.